### ISSUE TITLE
ANDI suggest use of aria-label when alt is present on input type text

### ISSUE THREAD

**Issue by mraccess77** [SSAgov/ANDI#16]
ANDI gives the following message "dangerText box has no accessible name, associated <label>, or [title].  warning[alt] attribute is meant for <img>, use [aria-label] on this element."

While alt is not valid and ANDI should flag that -- it should not suggest using aria-label and instead say provide an accessible name through properties allowed by the accessible name calculation such as label, title or ARIA.  Some agencies may want to use ANDI but may not agree on the recommendations provided by ANDI.

**Comment by SSAgov** [comment]
The additional suggestion text has been removed in ANDI 20.1.1. Now the alert message will read: Warning: [alt] attribute is meant for img. Thank you for the feedback.
